users/views.py

```diff
@@ -195,6 +195,11 @@ def movies(request):
         print("\nIntent is: {}".format(intent))
 
         # calling return answer funtion
+        answer_dict = {'get_top_hollywood_movies':['The Godfather','The Shawshank Redemption','The Lord of the Rings'],
```

Comment:
it is redundant, I have already setup the `answer_dict` just below the funs name.